script/merge_users

```diff
@@ -10,21 +10,26 @@ require_relative '../config/environment'
 first_user  = User.find_by_email!(first)
 second_user = User.find_by_email!(second)
 
-require 'highline'
-terminal = HighLine.new
-
-puts "Which email do you want to keep?"
-email = terminal.choose do |menu|
-  menu.choice first_user.email
-  menu.choice second_user.email
+def choose_from_two(first, second)
+  puts "Which handle do you want to keep? (1/2)"
```

Comment:
Since we are using it for email too, perhaps `which option..` or something similar would make more sense.

Reply:
Right, my bad.